dist/index.html

```diff
@@ -1,60 +1,85 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Block-Chain Summit</title>
-    <link rel="stylesheet"
-          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
     <link rel="stylesheet" href="./css/style.css">
+    <title>PBlock-Chain Expo</title>
 </head>
 <body>
-<header class="header-container">
+<header class="header-wrapper">
     <div id="myNav" class="overlay-nav">
         <a href="#" class="close_btn" >&times;</a>
-        <div class="overlay-nav-content">
+        <nav class="overlay-nav-content">
             <a href="./index.html">Home</a>
             <a href="./about.html">About</a>
             <a href="#">Program</a>
             <a href="#">Sponsor</a>
             <a href="#">Join</a>
             <a href="#">News</a>
-        </div>
-    </div>
-    <div class="header-container">
-        <nav class="nav-wrapper">
-            <div class="mobile-nav">
-                <div class="hamburger">
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                </div>
-            </div>
         </nav>
-        <div class="headline">
-            <h2>"Hello and Welcome!"</h2>
-            <h1>TECHEX </br> Blockchain-Expo 2023</h1>
-        </div>
-        <div class="headline-box">
-            <p>
-                This two day event will take place in November 2023, featuring 20
-                speakers and more than 500 participants. The theme for the two-day
-                Block-Chain Expo: Connecting the Blockchain Ecosystem
-            </p>
+    </div>
+    <div class="desktop-banner">
+        <div class="desktop-nav-banner">
+            <a href="#"><i class='fa fa-facebook'></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
```

Comment:
- [ ] As was requested by the previous reviewer, since this is a list of items. Kindly wrap the items in a `ul` tag and `li` to hold each list

Reply:
All requested changes fixed